curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md

```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --answer--
 
-Placeholder answer
+`background: red;`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you set the transparency of a color to 50%?
 
 #### --distractors--
 
-Placeholder distractor 1
+`bg-opacity: 0.5;`
 
 ---
 
-Placeholder distractor 2
+`transparency: 50%;`
 
 ---
 
-Placeholder distractor 3
+`alpha: 0.5;`
 
 #### --answer--
 
-Placeholder answer
+`opacity: 0.5;`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the equivalent `HSL` value for the color `#FF0000`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`hsl(180%, 100%, 50%)`
 
 ---
 
-Placeholder distractor 2
+`hsl(180, 100%, 50%)`
 
 ---
 
-Placeholder distractor 3
+`hsl(360, 100%, 50%)`
 
 #### --answer--
 
-Placeholder answer
+`hsl(0, 100%, 50%)`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you set the color of an element to a random color?
 
 #### --distractors--
 
-Placeholder distractor 1
+`color: random();`
 
 ---
 
-Placeholder distractor 2
+`color: #random;`
 
 ---
 
-Placeholder distractor 3
+`color: any();`
 
 #### --answer--
 
-Placeholder answer
+No direct way to set.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default value of alpha in `rgba` if it's not specified?
 
 #### --distractors--
 
-Placeholder distractor 1
+`0` (completely transparent)
 
 ---
 
-Placeholder distractor 2
+`2` (completely opaque)
 
 ---
 
-Placeholder distractor 3
+`0.5` (half transparent)
 
 #### --answer--
 
-Placeholder answer
+`1` (completely opaque)
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a valid hex color code?
 
 #### --distractors--
 
-Placeholder distractor 1
+`#12345G`
 
 ---
 
-Placeholder distractor 2
+`#12ffg0`
 
 ---
 
-Placeholder distractor 3
+`#12ffgg`
 
 #### --answer--
 
-Placeholder answer
+`#ff12ff`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How many hexadecimal characters are used to define a color in shorthand hex notation?
 
 #### --distractors--
 
-Placeholder distractor 1
+`5`
 
 ---
 
-Placeholder distractor 2
+`4`
 
 ---
 
-Placeholder distractor 3
+`2`
 
 #### --answer--
 
-Placeholder answer
+`3`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of these functions allows specifying the saturation of a color?
 
 #### --distractors--
 
-Placeholder distractor 1
+`rgb()`
 
 ---
 
-Placeholder distractor 2
+`hsl()`
 
 ---
 
-Placeholder distractor 3
+`rgba()`
 
 #### --answer--
 
-Placeholder answer
+`opacity()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following hex codes represents a shade of red?
 
 #### --distractors--
 
-Placeholder distractor 1
+`#0000ff`
 
 ---
 
-Placeholder distractor 2
+`#ffff00`
 
 ---
 
-Placeholder distractor 3
+`#00ff00`
 
 #### --answer--
 
-Placeholder answer
+`#ff0000`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which keyword sets the text color to the default browser color?
```

Comment:
Inherit does not set the value to the browser default. It sets the value to the computed value of the parent element. Which can sometimes be the default value, but not always.